themes/src/main/resources-community/theme/base/account/messages/messages_pt_BR.properties

```diff
@@ -293,9 +293,9 @@ authenticatorSMSMessage=A aplica\u00e7\u00e3o ir\u00e1 enviar o c\u00f3digo de v
 authenticatorSMSFinishSetUpMessage=As mensagens de texto ser\u00e3o enviadas para
 authenticatorDefaultStatus=Padr\u00e3o
 authenticatorChangePhone=Mudar N\u00famero de Celular
-authenticatorBackupCodesTitle=C\u00f3digos de Emerg\u00eancia
-authenticatorBackupCodesMessage=Veja seus c\u00f3digos de emerg\u00eancia de 8 d\u00edgitos
-authenticatorBackupCodesFinishSetUpMessage=12 c\u00f3digos de emerg\u00eancia foram gerados. Cada um pode ser utilizado apenas uma vez.
+authenticatorRecoveryAuthnCodesTitle=C\u00f3digos de Emerg\u00eancia
+authenticatorRecoveryAuthnCodesMessage=Veja seus c\u00f3digos de emerg\u00eancia de 12 d\u00edgitos
+authenticatorRecoveryAuthnCodesFinishSetUpMessage=12 c\u00f3digos de emerg\u00eancia foram gerados. Cada um pode ser utilizado apenas uma vez.
```

Comment:
I assume all the translations are just google translate or something? We don't usually add translations, but require the community to contribute those so they can be reviewed properly

Reply:
Hmm.. had no clue that we had ever had anything around backup codes. You sure these are not related to the old way of setting up OTP?